editor/app/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
     compileSdkVersion 28
     defaultConfig {
         applicationId "ca.ubc.eml.soiltopargraphy.editor"
-        minSdkVersion 15
+        minSdkVersion 24
```

Comment:
Why did you need to change the minSdk? It would be ideal to support more phones if possible.